source/cloud-native-platform/troubleshooting/index.html.md.erb

```diff
@@ -505,3 +507,103 @@ Check the pipeline passed after merging. If the pipeline failed, it's possible t
 Check the IP your rule is forwarding to. It should be the private IP of the frontend Application Gateway.
 
 You can find this [here](https://github.com/hmcts/azure-platform-terraform/blob/6f0b867e75b7e9cee9e7adc87084f6911eb5373d/environments/sbox/sbox.tfvars#L20).
+
+### Java (Spring Boot) Golden Path Errors
+
+#### - Dependencies security vulnerabilities can be resolved by updating the version
+
+We recommend to use the [OWASP Dependency Checker](https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/index.html).
+The checker provides monitoring of the project's dependencies and creates a report of known vulnerable components that are included in the build.
+
+#### Solution
+
+To resolve security vulnerabilities flagged by the OWASP Dependency Check plugin that can be resolved by upgrading dependencies,
+you can follow this chain of commands. The goal here is to update the flagged dependencies to their latest safe versions:
+
+Step 1: Run Dependency Check to Identify Vulnerabilities
+
+```shell
+./gradlew dependencyCheckAnalyze --info
+```
+
+Step 2: Review the Report
+After running the dependency check, open the generated report (e.g., build/reports/dependency-check-report.html) to
+identify the vulnerable dependencies and note their current versions.
```

Comment:
Might also make sense to advise on where they can find the report in Jenkins i.e. under Artifacts > build/reports/dependency-check-report.html